src/components/Jails/ReportingCounties/ReportingCounties.js

```diff
@@ -0,0 +1,67 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import Modal from "../../shared/Modal";
+import County from "../../shared/County";
+
+import "./ReportingCounties.scss";
+
+const ReportingCounties = ({ counties, stateName }) => {
+  const [open, setOpen] = useState(false);
+
+  return counties.length ? (
+    <span className="ReportingCounties">
+      See full list of counties&nbsp;
+      <button className="ReportingCounties__button" type="button" onClick={() => setOpen(!open)}>
+        here
```

Comment:
Reminder: according to the mocks, there should be a period after the word "here" that is not in the link.